src/buffer/out/textBuffer.cpp

```diff
@@ -2102,6 +2102,32 @@ std::wstring TextBuffer::GetPlainText(const CopyRequest& req) const
     return selectedText;
 }
 
+// Routine Description:
+// - Retrieves the text data from the buffer *with* ANSI escape code control sequences and presents it in a clipboard-ready format.

```

Comment:
In general, any guidance on comment max length? I didn't wrap any of the doc comments I added, they get a bit long heh 😅

Reply:
I try to stick to around 120 characters and 140 at most, because then you can still comfortably read it on smaller ~14" laptop screens. In this instance I'd probably split it into 2 lines and make them even length.

FYI you don't need to write "Routine Description:" on new functions. This is only a hold over from our old code base.